<launch>
  <node  pkg="move_base" type="move_base" respawn="false" name="move_base_node" output="screen">
    <remap from="odom" to="base_odometry/odom" />
    <remap from="cmd_vel" to="base_controller/command" />

    <rosparam file="$(find 3d_nav_executive)/config/sbpl_global_params.yaml" command="load" />
    <param name="SBPLLatticePlanner3D/primitive_filename" value="$(find sbpl_lattice_planner_3d)/pr2sides.mprim" />

    <rosparam file="$(find 3d_nav_executive)/config/pr2_footprint_links.yaml" command="load" ns="SBPLLatticePlanner3D" />


    <rosparam file="$(find 3d_nav_executive)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find 3d_nav_executive)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find 3d_nav_executive)/config/local_costmap_params_close.yaml" command="load" />
    <rosparam file="$(find 3d_nav_executive)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find 3d_nav_executive)/config/base_local_planner_trajectory.yaml" command="load" /> 
    <rosparam file="$(find 3d_nav_executive)/config/base_local_planner_poses.yaml" command="load" />
  </node>

</launch>
